packages/core/content-types/tiptap.js

```diff
@@ -42,7 +42,15 @@ export class Tiptap {
   }
 
   toHTML() {
-    return this.data;
+    let data = this.data.replaceAll(
+      'data-spacing="single"',
+      'style = "margin:0px"'
+    );
+    data = data.replaceAll(
+      'data-spacing="double"',
+      'style = "margin-bottom:0px"'
+    );
+    return data;
```

Comment:
this will be unnecessary because you are already changing the html template.

Reply:
yes we can do that.